apps/studio/next.config.mjs

```diff
@@ -25,7 +25,14 @@ const ContentSecurityPolicy = `
   object-src 'none';
   script-src 'self' 'unsafe-eval';
   style-src 'self' https: 'unsafe-inline';
-  connect-src 'self' https://schema.isomer.gov.sg https://browser-intake-datadoghq.com https://*.browser-intake-datadoghq.com https://vitals.vercel-insights.com/v1/vitals;
+  connect-src
+    'self'
+    https://schema.isomer.gov.sg
+    https://browser-intake-datadoghq.com
+    https://*.browser-intake-datadoghq.com
+    https://vitals.vercel-insights.com/v1/vitals
+    https://*.amazonaws.com
```

Comment:
should this be stricter? with s3.amazonaws?

Reply:
The hostname is like `bucket-name.s3.ap-southeast-1.amazonaws.com`, so the `connect-src` will need to be something like `*.s3.*.amazonaws.com` which is not supported because of the double wildcard.

Reply:
Realised that we already have the bucket-name and region information from the env vars so went ahead to use them inside the CSP. Updated in 89b2d6c9da1c8538c8fd9068966c81970308a10a.